code/game/jobs/job/medical.dm

```diff
@@ -59,17 +59,11 @@
 	selection_color = "#FF97D1"
 	economic_modifier = 7
 
-	minimum_character_age = 30
+	minimum_character_age = 25
```

Comment:
This is odd. The wiki rightfully points out that one can only have a medical degree at 28 years of age. Shouldn't that still be the case?